gcloud-java-compute/src/main/java/com/google/gcloud/spi/DefaultComputeRpc.java

```diff
@@ -43,4 +64,214 @@ public DefaultComputeRpc(ComputeOptions options) {
   private static ComputeException translate(IOException exception) {
     return new ComputeException(exception);
   }
+
+  @Override
+  public DiskType getDiskType(String zone, String diskType, Map<Option, ?> options)
+      throws ComputeException {
```

Comment:
No need for the explicit "throws ComputeException" (I see that in Storage we do that in the spi RPC interface and implementation but I think it is a leftover as originally it was IOException). In any case, for consistency I am fine with leaving it as is (here and in the spi interface), but it should be removed from the API interface (that has a javadoc to cover it).